app/assets/stylesheets/application.scss

```diff
@@ -11,6 +11,6 @@
  * It is generally better to create a new file per style scope.
  *
  */
-
- @import 'font-awesome';
- @import 'bootstrap';
+@import 'font-awesome';
+@import 'bootstrap';
+@import 'layout/**/*';
```

Comment:
I think it is a good practice to add an empty line when content of the file ends.